spec/models/card_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+RSpec.describe Card, type: "model" do
+describe Card
+  let(:card) { FactoryGirl.create(:card) }
+
+  it "has a valid factory" do
+    expect(card).to be_valid
+  end
+     
+  it "is invalid when original eq translate" do
+    card.translated_text = "Home"
+    expect(card).not_to be_valid
+  end
+
+  it "save downcase translate text" do
+    card.original_text = "ДоМ"
+    expect(card.translated_text).to eq("дом")
+  end
+
+describe "sort method" do
+ let(:card1) { FactoryGirl.create(:card, original_text: "Yellow",
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Use 2 (not 1) spaces for indentation.<br>Block body expression is on the same line as the block start.